great_expectations/render/components.py

```diff
@@ -742,11 +742,13 @@ def __init__(
         name: Union[str, AtomicDiagnosticRendererType, AtomicPrescriptiveRendererType],
         value: RenderedAtomicValue,
         value_type: Optional[str] = None,
+        exception: Optional[str] = None,
```

Comment:
What is this a `str` of? The class name? The error message?

Reply:
The error message. I'm not sure what class name you are referring to.